nautobot/docs/release-notes/version-1.3.md

```diff
@@ -64,6 +64,10 @@ Nautobot's REST API now supports multiple versions, which may requested by modif
 !!! tip
     As a best practice, when developing a Nautobot REST API integration, your client should _always_ request the current API version it is being developed against, rather than relying on the default API behavior (which may change with a new Nautobot major release, as noted, and which also may not include the latest and greatest API endpoints already available but not yet made default in the current release).
 
+#### Webhook Pre/Post-change Data Added to Request Body ([#330](https://github.com/nautobot/nautobot/issues/330))
+
+Webhooks now provide a snapshot of data before and after a change. This also includes the differences between the old and new data.
```

Comment:
Would probably be good to cross-link here to `models/extras/webhook.md` for additional details.